lxd/main_forkdns.go

```diff
@@ -1,9 +1,15 @@
 package main
 
 import (
+	"bufio"
+	"errors"
 	"fmt"
+	"os"
 	"strings"
 
+	"github.com/lxc/lxd/shared/dnsutil"
+	"github.com/lxc/lxd/shared/logger"
+	"github.com/lxc/lxd/shared/logging"
```

Comment:
Imports from our own packages go in their own block after import from external sources.